mne_bids/path.py

```diff
@@ -832,7 +832,7 @@ def _parse_ext(raw_fname, verbose=False):
     return fname, ext
 
 
-def get_entities_from_fname(fname):
+def get_entities_from_fname(fname, on_fail='raise'):
```

Comment:
can you rename it to on_error to match mne-python convention?

Reply:
Also did for _find_matching_sidecar.